2023-risks/m2-insecure-communication.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: col-sidebar
+title: "M2: Insecure Communication"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Most modern mobile applications exchange data with one or more remote servers. When the data transmission takes place, it typically goes through the mobile device's carrier network and the internet, a threat agent listening on the wire can intercept and modify the data if it transmitted in plaintext or using a deprecated encryption protocol. The following threat agents exist:
+
+- An adversary that shares your local network (compromised or monitored Wi-Fi);
+- Rogue carrier or network devices (routers, cell towers, proxy's, etc); or
+- Malware on your mobile device.
+
+# Attack Vectors	
+
+**Exploitability EASY**
+
+While modern applications do reply on cryptographic protocols such as SSL/TLS, they can sometimes have flaws in their implementations like: 
+
+- No hostname validation;
+- Accepting bad ssl certificates (self-signed, revoked, expired..); or
+- Inconsistency (having SSL/TLS only on select workflows such as authentication).
```

Comment:
+ outdated protocol versions
+ bad configuration, e.g., fallback to no encryption or outdated protocol